.github/workflows/actions.yml

```diff
@@ -19,8 +19,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.6"
```

Comment:
Let's add these back and this can be in a follow-up PR that drops 3.6 and 3.7. That way we can pin a major version to it.